webhook/server/server.go

```diff
@@ -163,7 +165,28 @@ func (s *WebhookServer) runAdmissionWebhookListenAndServe(handler http.Handler,
 			},
 		}
 
-		return secret, apply.WithOwner(secret).ApplyObjects(validatingWebhookConfiguration, mutatingWebhookConfiguration)
+		// Retry apply
```

Comment:
The change can improve resilience, but I am curious if the resolution addresses the original issue, which still seems unclear. 

In the current implementation, if the workloads fail to apply, there should be an error returned, and then longhorn-manager should fail to start. I assume that, eventually, the longhorn-manager pods should be started along with the webhook servers. 

[/webhook/webhook.go#L38-L40](https://github.com/longhorn/longhorn-manager/blob/a315536a02db2c7c2688ba77f4429b8564a4f3a2/webhook/webhook.go#L38-L40)
```go
		if err := s.ListenAndServe(); err != nil {
			logrus.Fatalf("Error %v webhook server failed: %v", webhookType, err)
		}
```

Reply:
Yes, I have the same thought, so the issue is addressed and mitigated in the second commit.